examples/with-docker/Dockerfile

```diff
@@ -22,6 +22,8 @@ WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 
+RUN mkdir -p public
```

Comment:
dont need to pass -p (only single folder)
and also I think dont need to write this mkdir command (and RUN mkdir -p .next/cache below) because "COPY --from=builder --chown=nextjs:nodejs" automatically create the folder for you